sentinel/src/db.js

```diff
@@ -55,6 +55,7 @@ if (UNIT_TEST_ENV) {
   module.exports.allDbs = stubMe('allDbs');
   module.exports.get = stubMe('get');
   module.exports.close = stubMe('close');
+  module.exports.medicDbName = stubMe('close');
```

Comment:
This doesn't seem right - `medicDbName` should be a string and therefore not have a `close` function?

Reply:
Whoops! The thing passed to stubMe is just the name that is used in the error message so I didn't notice. I'll fix it.